src/parameters/update_cost_parameters.jl

```diff
@@ -181,8 +182,9 @@ function update_variable_cost!(
     end
     mult_ = parameter_multiplier[component_name, time_period]
     expression = get_expression(container, FuelConsumptionExpression(), T)
-    cost_expr = expression[component_name, time_period] * fuel_cost * mult_
+    cost_expr = expression[component_name, time_period] * (fuel_cost * mult_)
     add_to_objective_variant_expression!(container, cost_expr)
+    # TODO: missing _mult below?
```

Comment:
no, one expression tracks the cost and the other the fuel consumption

Reply:
yup not much to do here, as the two need to be kept different